motionplan/solverFrame.go

```diff
@@ -260,7 +260,7 @@ func (sf *solverFrame) mapToSlice(inputMap map[string][]frame.Input) ([]frame.In
 	return inputs, nil
 }
 
-func (sf *solverFrame) sliceToMap(inputSlice []frame.Input) map[string][]frame.Input {
+func (sf *solverFrame) inputToMap(inputSlice []frame.Input) map[string][]frame.Input {
```

Comment:
It seems strange to me to have a private `inputToMap` and a public `InputFromMap` which both work on solverFrame pointers. I haven't read the big thread in `frame.go` yet, but I think some users would be annoyed to see it won't go both ways.

Reply:
Yeah, me and Peter have been discussing a bit what the best approach is for this so I'm going to wait on the outcome of that before making this change.